### PR TITLE
Phase3

### PR DESCRIPTION
**Pull request by mohamed99akram**
Added tabs, carousel, and top categories using bootstrap.
but the carousel is not responsive. and the search bar is now not working.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,99 +74,174 @@ <h1>Get ahead stay ahead</h1>
...
+                </div>
+                <div class="tab-pane fade" id="data-science" role="tabpanel" >
+                   c 
+                </div>
+                <div class="tab-pane fade" id="aws-certification" role="tabpanel">
+                  v  
+                </div>
+                <div class="tab-pane fade" id="drawing" role="tabpanel" >
+                 n   
+                </div>
```
Comment (ahmed2m) [inline index.html]: Better to just leave the div empty if you're going to replace contents on load anyway

File: script.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        `
+            <div class="course carousel-item">
+                ${getCourseCard(course)}
+            </div>
+            `;
+    return html;
+}
+let currentTab = "Python";
+function updateCourses(courses){
+    console.log("hi how are you?");
```
Comment (ahmed2m) [inline script.js]: best to remove any debugging logging

File: script.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    let coursesContainer = document.querySelector("#myTabContent");
+    courses.forEach((course) => coursesContainer.innerHTML += getCourseCard(course));
```
Comment (ahmed2m) [inline script.js]: Search is working! But the styles are not. Let's have a small meeting to work on it together

**Review by ahmed2m — CHANGES_REQUESTED** [review]

File: script.js (hunk 3 above)
Comment (mohamed99akram) [reply script.js]: Ok, will the meeting be during the group meeting?

Comment (ahmed2m) [reply script.js]: Can do it right now, check slack

**Review by ahmed2m — APPROVED** [review]

**Pull request merged**